cmake/CMakeLists.txt

```diff
@@ -108,8 +108,10 @@ option(onnxruntime_USE_TELEMETRY "Build with Telemetry" OFF)
 #It's default OFF because it's experimental now.
 option(onnxruntime_PREFER_SYSTEM_LIB "Experimental: Build with the preinstalled libraries in your system" OFF)
 
+# ORT mininal build settings
 option(onnxruntime_MINIMAL_BUILD "Exclude as much as possible from the build. Support ORT format models. No support for ONNX format models." OFF)
 option(onnxruntime_REDUCED_OPS_BUILD "Reduced set of kernels are registered in build via modification of the kernel registration source files." OFF)
+option(onnxruntime_NO_EXCEPTIONS "Disable exception handling." OFF)
```

Comment:
Would be good to add to the comment saying this will only work in a minimal build currently. would need ONNX changes for it to work standalone.